pkg/api/const.go

```diff
@@ -25,14 +25,16 @@ const (
 
 // Distro string consts
 const (
-	Ubuntu          Distro = "ubuntu"
-	Ubuntu1804      Distro = "ubuntu-18.04"
-	RHEL            Distro = "rhel"
-	CoreOS          Distro = "coreos"
-	AKS             Distro = "aks"
-	AKSDockerEngine Distro = "aks-docker-engine" // deprecated docker-engine distro
-	AKS1804         Distro = "aks-18.04"
-	ACC1604         Distro = "acc-16.04"
+	Ubuntu            Distro = "ubuntu"
+	Ubuntu1804        Distro = "ubuntu-18.04"
+	RHEL              Distro = "rhel"
```

Comment:
Adding a version component to consts `rhel` and `coreos` may fit this PR

Reply:
I don't think that's in the scope of this PR, I would rather leave the non-Ubuntu OSes untouched. Also, we use versions `latest` for both of these.